src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -2318,8 +2318,25 @@ void func_80833A20(Player* this, s32 newMeleeWeaponState) {
     this->meleeWeaponState = newMeleeWeaponState;
 }
 
-s32 func_80833B2C(Player* this) {
-    if (this->stateFlags1 & (PLAYER_STATE1_16 | PLAYER_STATE1_PARALLEL | PLAYER_STATE1_LOCK_ON_FORCED_TO_RELEASE)) {
+/**
+ * This function checks for friendly (non-hostile) Z-Target related states.
+ * For hostile related lock-on states, see `Player_UpdateHostileLockOn` and `Player_CheckHostileLockOn`.
+ *
+ * Note that `PLAYER_STATE1_FRIENDLY_ACTOR_FOCUS` will include all `focusActor` use cases that relate to
+ * friendly actors. This function can return true when talking to an actor, for example.
+ * Despite that, this function is only releveant in the context of actor lock-on, which is why the function
+ * name only mentions "friendly lock-on".
```

Comment:
this comment is misleading, as the function name also mentions "Parallel", plus the comment says it's only relevant in the context of lock-on, which Parallel is separate from

Reply:
im speaking particulalry to the lock-on vs focus dynamic here, which is what the whole paragraph is about. Ill try to think of something to make that more clear but suggestions are welcome

Reply:
Oh I see, okay, that makes sense then. I wasn't getting that you meant just lock on vs focus. also `releveant` 😇

Reply:
lol one of those words i always spell wrong, thanks. if there is anything i can add to make it clear lmk

Reply:
tried something, lmk

Reply:
yeah that makes it clear to me! definitely better.